scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -112,6 +112,18 @@ object Term {
   @ast class Match(expr: Term, cases: List[Case] @nonEmpty) extends Term
   @ast class Try(expr: Term, catchp: List[Case], finallyp: Option[Term]) extends Term
   @ast class TryWithHandler(expr: Term, catchp: Term, finallyp: Option[Term]) extends Term
+  @ast class ContextFunction(params: List[Term.Param], body: Term) extends Term {
+    checkFields(
+      params.forall(param =>
+        param.is[Term.Param.Quasi] ||
+          (param.name.is[scala.meta.Name.Anonymous] ==> param.default.isEmpty)
+      )
+    )
+    checkFields(
+      params.exists(_.is[Term.Param.Quasi]) ||
+        params.exists(_.mods.exists(_.is[Mod.Implicit])) ==> (params.length == 1)
```

Comment:
Should it not be Mod.Using ?

Reply:
copy pasted it from `Term.Function` it should actually not have implicit/using mods at all as `?=>` implicitly says it is using (explicitly?)

Reply:
from docs:
`g(ExecutionContext ?=> f(3))  // is expanded to g((using ev: ExecutionContext) => f(3)(using ev))`